service/src/main/resources/changelog/changesets/20230118_assemble_refbased_prepopulation.yaml

```diff
@@ -0,0 +1,1597 @@
+databaseChangeLog:
+  - changeSet:
+      id: "method prepopulation for assemble_refbased"
+      author: kpierre
+      changes:
+        - insert:
+            tableName: method
+            columns:
+              - column:
+                  name: method_id
+                  value: 00000000-0000-0000-0000-000000000005
+              - column:
+                  name: name
+                  value: assemble_refbased
+              - column:
+                  name: description
+                  value: assemble_refbased
+              - column:
+                  name: created
+                  valueDate: "CURRENT_TIMESTAMP()"
+              - column:
+                  name: method_source
+                  value: Github
+              - column:
+                  name: last_run_set_id
+                  value: null
+        - insert:
+            tableName: method_version
+            columns:
+              - column:
+                  name: method_version_id
+                  value: 50000000-0000-0000-0000-000000000005
+              - column:
+                  name: method_id
+                  value: 00000000-0000-0000-0000-000000000005
+              - column:
+                  name: method_version_name
+                  value: '1.0'
+              - column:
+                  name: method_version_description
+                  value: 'Target workflow assemble_refbased'
+              - column:
+                  name: method_version_url
+                  value: 'https://raw.githubusercontent.com/broadinstitute/viral-pipelines/master/pipes/WDL/workflows/assemble_refbased.wdl'
+        - insert:
+            tableName: run_set
+            columns:
+              - column:
+                  name: run_set_id
+                  value: 10000000-0000-0000-0000-000000000005
+              - column:
+                  name: method_version_id
+                  value: 50000000-0000-0000-0000-000000000005
+              - column:
+                  name: is_template
+                  value: true
+              - column:
+                  name: run_set_name
+                  value: assemble_refbased workflow
+              - column:
+                  name: run_set_description
+                  value: "assemble_refbased workflow"
```

Comment:
Same comment here. Would it be better to call it `Example assemble_refbased workflow` or `Sample assemble_refbased workflow`?

Reply:
My vote would be for `assemble_refbased sample submission` Ideally we will be hiding these from actual user view so it should be moot, but in case it leaks out somehow, that feels most "accurate" to me